packages/caravan-bitcoin/src/fixtures.ts

```diff
@@ -1162,9 +1162,9 @@ const MULTISIGS_BASE = [
r5pawnggbo+MwtVaL+////sOe96h5S5rxHEXLGt4smpzgpEbWRC1yUftGJg74TaEUBAAAAFxYAFEJF2DhyeBgej11h41Qn+gVfiRzO/v///7aPh/KPEXELcAVCY0DExNl5XDMfAnVJvTg7bxedcJooAAAAAAD+////edA3RH0HBggyEuwWils+JPl716ngCZsQWwQngYU3gIEAAAAAFxYAFOxiocogCr3HVcizT6rnggK23T/D/v///wKSBQAAAAAAABepFEnpEzqv+pGS2AJlXy24+h257sAIhxAnAAAAAAAAF6kUGrz4zqMhyodN5L60+XUHf+hkpUqHAkcwRAIgacCYUWTRfHRvuBgXn5aXQOOdZWnsWhRKg8ct4kc7a4ACIE9iIXu7KzTv7dpFTIXfqWB4jUx/U4x/3yLFB09aLHi7ASED5091fOsCiKx8BRt9uH+wWaGerAWkgMZckTBopvgNI7gCRzBEAiAPgntk5f62Bx9dQnjk+jZD4WQxariX9T3wSCZY51MRAgIgEUP0NIgSEiMih2qimnKFzI6lGj4y+wOS/S5kItY3k/UBIQNQ3lNpPJ2n+EnKuUeTBO4rKvMX8o7gDtHhEs2VvCAL0gJHMEQCIAJeDv8JsJgX6NWVwBv9L9umCC4sT48oZUX5oqLp8Ui6AiBXJXBOFMKWmhr88AhN0pvbxRNrdUOBWOz3aZLjwktWHwEhAuJOffJgy1b/vNtSvqb4yLwmfYSfV5nPkZ4yV0Vo7YXWAkcwRAIgEbvZ7GM48XT6Cjhqyx+03+NINDsu7VAXsR6WFRrSFMoCIH57tUQ/NNMxR/KPr7TNZh3Mx8aZOtfFjtRxl8hbSMhJASECWHmQxfcdnkyJQGlYmEXF8mSbgwBhYLnRuPMbq1QTOpOZSQkAAQEgECcAAAAAAAAXqRQavPjOoyHKh03kvrT5dQd/6GSlSocBBCIAIASdbpRQdFJbA+BId1k2j/Zj8Qu4iXYBe92dPM6EkIXlAQVHUiEDKLV8L2XJjtfN5LylTMOhOvpNRxF/2drgZmOkFp4F74YhA0KZf2/Nf6Sjx+KQyIZxSJkuYZR0ISCYXGZNniFEYa98Uq4iBgMotXwvZcmO183kvKVMw6E6+k1HEX/Z2uBmY6QWngXvhhwAAAAFMAAAgAAAAIBkAACAAQAAgAAAAAAAAAAAIgYDQpl/b81/pKPH4pDIhnFImS5hlHQhIJhcZk2eIURhr3wc9X7GXTAAAIAAAACAZAAAgAEAAIAAAAAAAAAAAAABAPcCAAAAAAEBpc1RFIj9v1ZibEYhV0GzCVa0f43XQWSg2qXxo+EdwXQBAAAAFxYAFO7v8OENopc0STMankjyRrGEiOll/v///wIQJwAAAAAAABepFBq8+M6jIcqHTeS+tPl1B3/oZKVKh70qAAAAAAAAF6kURad5R89C23/sQc7CXCivC3O+zy6HAkcwRAIgRIWKhxF/2K8vqS2xI5NQGddu6RV/uaSKnNl+4IWxKPkCIFcreZ6ubz+M9qoodFD/RrIViIQNq+62PlfG8/Mg9J7qASECWr2YLwyeu7cjiQAyavQ3w5PrLF6HX1vWMVLd11blBJqZSQkAAQEgECcAAAAAAAAXqRQavPjOoyHKh03kvrT5dQd/6GSlSocBBCIAIASdbpRQdFJbA+BId1k2j/Zj8Qu4iXYBe92dPM6EkIXlAQVHUiEDKLV8L2XJjtfN5LylTMOhOvpNRxF/2drgZmOkFp4F74YhA0KZf2/Nf6Sjx+KQyIZxSJkuYZR0ISCYXGZNniFEYa98Uq4iBgMotXwvZcmO183kvKVMw6E6+k1HEX/Z2uBmY6QWngXvhhwAAAAFMAAAgAAAAIBkAACAAQAAgAAAAAAAAAAAIgYDQpl/b81/pKPH4pDIhnFImS5hlHQhIJhcZk2eIURhr3wc9X7GXTAAAIAAAACAZAAAgAEAAIAAAAAAAAAAAAABAPcCAAAAAAEB1bu9SV5b/QDfMtnimjtFZmVfQfBwx609+WvkNuBkv1sAAAAAFxYAFA4CHBwzE++ZGs0MXZ9jyhLOEQAF/v///wKr1gEAAAAAABepFF0n21jxPUhRJgF02dyerN9iB0pOhxAnAAAAAAAAF6kUGrz4zqMhyodN5L60+XUHf+hkpUqHAkcwRAIgd7Tw8tFIBEP4sp7Wiim2Brg6PrvaQ0m/gTwzMwt/SqkCICBhgj/SUzC2wK22ivzkVGraGT62OYLyinVivGBeprDlASECkwp7im/VGozjYDm6zxYCOc+328ctwaBRqqwzO8f455JOSQkAAQEgECcAAAAAAAAXqRQavPjOoyHKh03kvrT5dQd/6GSlSocBBCIAIASdbpRQdFJbA+BId1k2j/Zj8Qu4iXYBe92dPM6EkIXlAQVHUiEDKLV8L2XJjtfN5LylTMOhOvpNRxF/2drgZmOkFp4F74YhA0KZf2/Nf6Sjx+KQyIZxSJkuYZR0ISCYXGZNniFEYa98Uq4iBgMotXwvZcmO183kvKVMw6E6+k1HEX/Z2uBmY6QWngXvhhwAAAAFMAAAgAAAAIBkAACAAQAAgAAAAAAAAAAAIgYDQpl/b81/pKPH4pDIhnFImS5hlHQhIJhcZk2eIURhr3wc9X7GXTAAAIAAAACAZAAAgAEAAIAAAAAAAAAAAAAAAQAiACBYHpgGYQrByCSdw0RuAfg2Xs/FBkYjedaUi6oNEoxRmgEBR1IhAnKDT2UkRJEOHMFXuZ9S/WBSgmoR+QmYeaVUkPa4GQvnIQKZl/nHCUiB0kGxOxTYVnze3KzNQRhExV/gmwuMobJUsVKuIgICcoNPZSREkQ4cwVe5n1L9YFKCahH5CZh5pVSQ9rgZC+ccAAAABTAAAIAAAACAZAAAgAEAAIABAAAAAAAAACICApmX+ccJSIHSQbE7FNhWfN7crM1BGETFX+CbC4yhslSxHPV+xl0wAACAAAAAgGQAAIABAACAAQAAAAAAAAAA",
```

Comment:
That's a big 'un!
<img width="1358" alt="Screenshot 2024-09-08 at 1 00 37 PM" src="https://github.com/user-attachments/assets/83d3617a-c65f-4082-b23c-06d13b98332c">

<img width="1380" alt="Screenshot 2024-09-08 at 1 00 50 PM" src="https://github.com/user-attachments/assets/adcce7b8-e7fd-435e-9508-019c751462ac">